data_model.mdd

```diff
@@ -355,13 +358,27 @@
         bool enable_notification "Parental leave or career breaks require notifications if they are within 30days"
     }
     SupportRecord {
-        int id
+        int id PK
         int trainee_id FK
-        date date "Support Record Date Entered onto DB, Null: True"
+        int case_type_id FK "Support Issue Type, Null: True"
+        date open_date "Support Record Date Entered onto DB, Null: True"
```

Comment:
The open date shouldn't be null=True, as this is a new record we don't need to worry about inconsistent data here, it should just be the current date the time of save.

Reply:
ah, great idea re current date, nullability was mainly defensive against incomplete data imports

plus I am enforcing open date on the UI at record creation. will remove nullability